build_tools/install_rocm_from_artifacts.py

```diff
@@ -99,19 +94,20 @@ def _get_github_release_assets(release_tag, amdgpu_family, release_version):
     if gh_token:
         headers["Authentication"] = f"Bearer {gh_token}"
 
-    response = requests.get(github_release_url, headers=headers)
-    if response.status_code == 403:
-        log(
-            f"Error when retrieving GitHub release assets for release tag '{release_tag}'. This is most likely a rate limiting issue, so please try again"
-        )
-        return
-    elif response.status_code != 200:
-        log(
-            f"Error when retrieving GitHub release assets for release tag '{release_tag}'. Exiting..."
-        )
-        return
+    request = Request(github_release_url, headers=headers)
+    with urlopen(request) as response:
+        if response.status == 403:
+            log(
+                f"Error when retrieving GitHub release assets for release tag '{release_tag}'. This is most likely a rate limiting issue, so please try again"
+            )
+            return
+        elif response.status != 200:
+            log(
+                f"Error when retrieving GitHub release assets for release tag '{release_tag}'. Exiting..."
+            )
+            return
```

Comment:
Also log the `response.status` when != 200, and maybe any additional error messages on the request.